core/src/commonMain/kotlin/com/xebia/functional/xef/metrics/LogsMetric.kt

```diff
@@ -46,5 +46,9 @@ class LogsMetric : Metric {
     logger.info { "${writeIndent(numberOfBlocks.get())}|-- $key = $value" }
   }
 
+  override suspend fun parameter(key: String, values: List<String>) {
+    logger.info { "${writeIndent(numberOfBlocks.get())}|-- $key = $values" }
```

Comment:
Should we mention here that it's about a `parameter` in the log message?

**off-topic:** should we take a `LogLevel` as a parameter in `LogsMetric`?

Reply:
Let me think about it. We could probably address this in a separate pull request.